.changeset/flat-flies-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/layer2-sequencer-health-adapter': minor
+---
+
+Add patch for the ethereum-cryptopgraph package so that the EA can successfully build using the StarknetJS dependency
```

Comment:
Should probably also mention support for Starkware?

Reply:
Added